### ISSUE TITLE
Add support for IntelliJ IDEA 2020.3 EAP 

### ISSUE THREAD

**Issue by sureshg** [suusan2go/kotlin-fill-class#42]
Thanks for this nice plugin. The plugin doesn't work on 2020.3 EAP. Please add the support for EAP versions.

```
IntelliJ IDEA 2020.3 EAP (Ultimate Edition)
Build #IU-203.5251.39, built on October 23, 2020
IntelliJ IDEA EAP User
Expiration date: November 22, 2020
Runtime version: 11.0.8+10-b1145.3 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.7
```

**Comment by Zyarra** [comment]
Yes please. I love the idea of this plugin..

**Comment by suusan2go** [comment]
I'm sorry for the late reply, I just published v1.0.5 which supports IDEA 2020.3. it will be available within 2 or 3 days.

**Comment by Zyarra** [comment]
> I'm sorry for the late reply, I just published v1.0.5 which supports IDEA 2020.3. it will be available within 2 or 3 days.

Youre the best ! ;)
